GetIndexNamesOrchestrator/index.js

```diff
@@ -0,0 +1,14 @@
+const df = require('durable-functions');
+
+module.exports = df.orchestrator(function* orchestratorFunctionGenerator(context) {
+  context.log('Starting GetIndexNamesOrchestrator');
+  const apimApiName = context.df.getInput();
+
+  context.log({ apimApiName });
+
+  const indexNames = yield context.df.callActivity('GetIndexNames', apimApiName);
+
+  context.log({ indexNames });
```

Comment:
Could some additional information be added to increase the usefulness of this message within the logs please?